ambry-protocol/src/test/java/com.github.ambry.protocol/RequestResponseTest.java

```diff
@@ -384,12 +384,15 @@ public void adminRequestResponseTest() throws IOException {
       requestStream = serAndPrepForRead(adminRequest, -1, true);
       deserAdminRequestAndVerify(requestStream, clusterMap, correlationId, clientId, type, null);
       // response
-      AdminResponse response = new AdminResponse(correlationId, clientId, ServerErrorCode.No_Error);
+      ServerErrorCode[] values = ServerErrorCode.values();
```

Comment:
minor: can we do No_Error and one random error. Don't want to skip on No_Error.

Reply:
Not sure I understand - how is "No_Error" diff from any other error code as far as this class and piece of code is concerned? It's not actually doing anything based on the error.

Reply:
got you. You can ignore my comment then